gui/slick/js/ajaxEpSearch.js

```diff
@@ -1,4 +1,4 @@
-var searchStatusUrl = srRoot + '/home/getManualSearchStatus';
+var searchStatusUrl = 'home/getManualSearchStatus';
```

Comment:
@OmgImAlexis are you sure this will work ? we had to revert something related "ui clenaup" because it make images and other stuff not to work

https://github.com/pymedusa/SickRage/pull/823/commits/7e717393f596179b27824fe5d803329b26800b5c

@medariox @Thraxis  can explain better